### PR TITLE
Added support for class path jars to be in a 'libs' folder in the wor…

### PR DESCRIPTION
**Pull request by jeremylcarter**
Enables support for a 'libs' folder in the workspace root directory that contains jars that will be added to the classpath.

Happy to discuss other options and have this PR declined as this is a bit of a hack.

### CODE REVIEW

**Comment by fwcd** [comment]
@jeremylcarter Thanks for your contribution, I like the idea and will definitely not reject it.

Still, instead of hardcoding a directory name, there could be a project-specific configuration file that contains the folder name(s). This would fit better in the existing architecture too:

https://github.com/fwcd/KotlinLanguageServer/blob/5590248efc356f4d4b8a551d385c0692a79ea642/src/main/kotlin/org/javacs/kt/classpath/findClassPath.kt#L38-L48

File: src/main/kotlin/org/javacs/kt/classpath/findClassPath.kt [hunk 1]
```diff
@@ -19,11 +19,19 @@ import java.util.Comparator
 import java.util.concurrent.TimeUnit
 
 fun findClassPath(workspaceRoots: Collection<Path>): Set<Path> {
-    return workspaceRoots
+    return ensureStdLibInPaths(workspaceRoots
             .flatMap { projectFiles(it) }
             .flatMap { readProjectFile(it) }
             .toSet()
             .ifEmpty { backupClassPath() }
```
Comment (fwcd) [inline src/main/kotlin/org/javacs/kt/classpath/findClassPath.kt]: We could remove this line as its functionality is already provided by `ensureStdLibinPaths`

Comment (jeremylcarter) [reply src/main/kotlin/org/javacs/kt/classpath/findClassPath.kt]: Good pick up 👍

**Comment by fwcd** [comment]
@jeremylcarter I would suggest [TOML](https://en.wikipedia.org/wiki/TOML) as a clean and lightweight format:

```toml
[classpath]
jars = [
    "folderWithJars/*",
    "library.jar",
    "another-library.jar"
]
```

This config file would be intended for small projects which use neither Maven nor Gradle and could easily be extended to contain other metadata if needed.

A TOML parser like [toml4j](https://github.com/mwanji/toml4j) could be integrated by adding the following line to the dependencies section in `build.gradle`:

```gradle
implementation "com.moandjiezana.toml:toml4j:0.7.2"
```

**Comment by jeremylcarter** [comment]
I'll be implementing support for TOML. I'll close this PR.

**Pull request closed without merging**